tinydrive/tinydrive-api/standalone.md

```diff
@@ -63,6 +114,14 @@ The `tinydrive.browse` method enables you to browse your files stored in {{site.
 
 {% include live-demo.html id="drive-standalone-browse" type="tinydrive" %}
 
+## `tinydrive.controls.browse`
+
+The `tinydrive.controls.browse` method enables you to browse your files stored in {{site.cloudfilemanager}} but without the possibility to pick them to be inserted. This might be useful if you want to use {{site.cloudfilemanager}} as a generic file manager.  A [Control Object](#controlobject) is returned which control the {{site.cloudfilemanager}} lifecycle, which it no longer does by itself. For a complete list of available settings to pass in to this method see [Picker settings](#pickersettings).
```

Comment:
```suggestion
The `tinydrive.controls.browse` method enables you to browse your files stored in {{site.cloudfilemanager}} but without the possibility to pick them to be inserted. This might be useful if you want to use {{site.cloudfilemanager}} as a generic file manager.  A [Control Object](#controlobject) is returned which control the {{site.cloudfilemanager}} lifecycle, which is not handled by {{site.cloudfilemanager}} when using the `controls` APIs. For a complete list of available settings to pass in to this method, see: [Picker settings](#pickersettings).
```